templates.json

```diff
@@ -2,39 +2,124 @@
+      "Name": "Alfentanil",
+      "Url": "https://raw.githubusercontent.com/Open-Systems-Pharmacology/Alfentanil-Model/v2.2/Alfentanil-Model.json"
     },
     {
       "Type": "Compound",
-      "Name": "Rifampicin",
-      "Url": "https://raw.githubusercontent.com/Open-Systems-Pharmacology/Rifampicin-Model/v1.1/Rifampicin-Model.json"
+      "Name": "Alprazolam",
+      "Url": "https://raw.githubusercontent.com/Open-Systems-Pharmacology/Alprazolam-Model/v1.1/Alprazolam-Model.json"
+    },
+    {
+      "Type": "Compound",
+      "Name": "Amikacin",
+      "Url": "https://raw.githubusercontent.com/Open-Systems-Pharmacology/Amikacin-Model/v1.2/Amikacin.json"
+    },
+    {
+      "Type": "Compound",
+      "Name": "Atazanavir",
+      "Url": "https://raw.githubusercontent.com/Open-Systems-Pharmacology/Atazanavir-Model/v1.1/Atazanavir-Model.json"
     },
     {
       "Type": "Compound",
       "Name": "Cimetidine",
-      "Url": "https://raw.githubusercontent.com/Open-Systems-Pharmacology/Cimetidine-Model/v1.0/Cimetidine-Model.json"
+      "Url": "https://raw.githubusercontent.com/Open-Systems-Pharmacology/Cimetidine-Model/v1.1/Cimetidine-Model.json"
+    },
+    {
+      "Type": "Compound",
+      "Name": "Clarithromycin",
+      "Url": "https://raw.githubusercontent.com/Open-Systems-Pharmacology/Clarithromycin-Model/v1.2/Clarithromycin-Model.json"
+    },
+    {
+      "Type": "Compound",
+      "Name": "Dapagliflozin",
+      "Url": "https://raw.githubusercontent.com/Open-Systems-Pharmacology/Dapagliflozin-Model/v1.1/Dapagliflozin-Model.json"
+    },
+    {
+      "Type": "Compound",
+      "Name": "Efavirenz",
+      "Url": "https://raw.githubusercontent.com/Open-Systems-Pharmacology/Efavirenz-Model/v1.1/Efavirenz-Model.json"
+    },
+    {
+      "Type": "Compound",
+      "Name": "Erythromycin",
+      "Url": "https://raw.githubusercontent.com/Open-Systems-Pharmacology/Erythromycin-Model/v1.2/Erythromycin-Model.json"
+    },
+    {
+      "Type": "Compound",
+      "Name": "Fluconazole",
+      "Url": "https://raw.githubusercontent.com/Open-Systems-Pharmacology/Fluconazole-Model/v1.0/Fluconazole-Model.json"
+    },
+    {
+      "Type": "Compound",
+      "Name": "Fluvoxamine",
+      "Url": "https://raw.githubusercontent.com/Open-Systems-Pharmacology/Fluvoxamine-Model/v1.1/Fluvoxamine-Model.json"
     },
     {
       "Type": "Compound",
       "Name": "Itraconazole",
-      "Url": "https://raw.githubusercontent.com/Open-Systems-Pharmacology/Itraconazole-Model/v1.0/Itraconazole-Model.json"
+      "Url": "https://raw.githubusercontent.com/Open-Systems-Pharmacology/Itraconazole-Model/v1.3/Itraconazole-Model.json"
     },
     {
       "Type": "Compound",
       "Name": "Hydroxy-Itraconazole",
-      "Url": "https://raw.githubusercontent.com/Open-Systems-Pharmacology/Itraconazole-Model/v1.0/Itraconazole-Model.json"
+      "Url": "https://raw.githubusercontent.com/Open-Systems-Pharmacology/Itraconazole-Model/v1.3/Itraconazole-Model.json"

```

Comment:
@msevestre I forgot what we said: do we need to mark the metabolites in the master templates file?
(e.g. that **Hydroxy-Itraconazole** is a metabolite of **Itraconazole**)